README.md

```diff
@@ -130,7 +130,7 @@ Invalid input or error
 Error: path/to/input Error Message
 ```
 
-If the input is invalid in any way, validate_io will sys.exit and throw an exception which can be detected by Nextflow and handled accordingly.
+If the input is invalid in any way, __main__ will sys.exit and throw an exception which can be detected by Nextflow and handled accordingly.
 
 ## references:
 Initial design: https://uclahs.box.com/s/eejwmwmdky7wsfcrs8a3jijy70rh6atp
```

Comment:
Not for this PR -

This URL isn't publicly accessible. I suggest we either remove the reference, or make the content available elsewhere. In the GitHub wiki?